### PR TITLE
refactor: Variables and helpers

### LINKED ISSUE #10: Use Variables across components
- refactor and fix the components to use the variables

### PR DESCRIPTION
**Pull request by shivampandey0**
- added variables for units
- refactored all the components to use variables.

### CODE REVIEW

File: css/helpers/flexbox.css [hunk 1]
```diff
@@ -12,17 +12,30 @@
...
   justify-content: center;
   align-items: center;
 }
 
-
 /* Justify */
-.justify-sb{
+.justify-sb {
   justify-content: space-between;
 }
```
Comment (shuklahimanshu81) [inline css/helpers/flexbox.css:23]: It's better to  use margin-left : auto;

**Review by shuklahimanshu81 — COMMENTED** [review]
Everything else looks good to me.

File: css/helpers/flexbox.css (hunk 1 above)
Comment (shivampandey0) [reply css/helpers/flexbox.css:23]: It is a helper-class specific for flexbox.

**Comment by saiyerniakhil** [comment]
Good one buddy, extensive availability of helper classes and variables reminds me of tailwind.css. Looks good to be merged ☄️

File: css/components/badges.css [hunk 2]
```diff
@@ -45,24 +49,89 @@
...
 }
 
-.badge-wrapper .avatar-xs ~ .badge {
-  right: 6px;
-  bottom: 12px;
-  width: 0.75rem;
-  height: 0.75rem;
+.avatar-xs ~ .badge.bottom-right {
+  bottom: calc(var(--size-1) / 2);
+  right: calc(var(--size-1) - 8px);
```
Comment (saiyerniakhil) [inline css/components/badges.css:114]: any specific purpose for not using --size-2 instead of `8px`?

Comment (shivampandey0) [reply css/components/badges.css:114]: not specific but wanted to use only one variable here.

**Pull request merged**